### ISSUE TITLE
Incorrect behavior of NVDA when interacting with icons in the system tray

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#9461]
### Steps to reproduce:
1. Launch NVDA.
2. Install first if not installed and launch if not already running the desktop version of "OneDrive". It is required to reliably reproduce the issue.
3. Press Windows+B to move the system focus to the system tray.
4. Use UP/DOWN arrows to move between the icons in the system tray.
5. Go to the icon of "OneDrive" and try to interact with it via the keyboard (SPACEBAR for single left mouse click, ENTER for double left mouse click, APPLICATIONS for single right mouse click). You'll notice, that nothing happens.
6. Use another screen reader (e.g. Narrator) to do the steps from 3 to 5. You'll notice, that the interaction works as expected.
7. Observe how when the steps are performed with NVDA, the mouse cursor gets moved to the system tray and to each icon in it, when the focus is moved there and between the icons there. When Narrator for example is used, this does not occur.

### Actual behavior:
Interaction with some icons in the system tray (e.g. the one for "OneDrive") fails. I think the cause is the movement of the mouse cursor when the interaction is done with NVDA.

### Expected behavior:
The mouse cursor should not be moved to the system tray and its icons, when NVDA interacts with the system tray and its icons.

### System configuration
- Operating system: Windows 10 Pro version 1809 (build 17763.404), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-17012,f3b9c3ce, installed, in bulgarian, with all add-ons disabled.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA?
Yes, the described behavior seams to be like this for a long time, but until recently I thought that it was a standard Windows behavior, until I've noticed that it does not occur with other screen readers like Narrator.

**Comment by DataTriny** [comment]
This behavior seems to be here on purpose, here is what I found in the explorer app module:

```python
if mouseHandler.lastMouseEventTime < time.time() - 0.2:
	# This focus change was not caused by a mouse event.
	# If the mouse is on another toolbar control, the notification area toolbar will rudely
	# bounce the focus back to the object under the mouse after a brief pause.
	# Moving the mouse to the focus object isn't a good solution because
	# sometimes, the focus can't be moved away from the object under the mouse.
	# Therefore, move the mouse out of the way.
	winUser.setCursorPos(0, 0)
```

I confirm that OneDrive's tray icon can be activated via the keyboard if NVDA is closed (Windows 10 version 1809, NVDA 2019.1).

**Comment by Brian1Gaff** [comment]
This is a repeat of one a few weeks back if I'm not mistaken.
 I'm not sure but I think its possibly the active state of the one drive 
icon which is the cause of this on a screenreader.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by DrSooom** [comment]
@DataTriny: Thanks a lot for this hint. I didn't recognize this until now. It's the same mouse movement on Win7x64 after the Systray got focus. I opened issue #9462 regarding this right now.

**Comment by Adriani90** [comment]
@k-kolev1985 are you still having this issue? I am using the most current version of One Drive with NVDA 2023.1 RC1 and cannot reproduce the issue. However, I am using Windows 11 22 H2 update.

**Comment by k-kolev1985** [comment]
@Adriani90 I just checked and no - I can't reproduce it anymore. I'm also using Windows 11 version 22H2 (build 22621.1413).

**Comment by lukaszgo1** [comment]
This issue still occurs on Windows 10 22H2  and any version of Windows older than Windows 11. Since it is caused by NVDA I'd like to suggest marking it as P4.
